lib/WorkerHandler.js

```diff
@@ -6,6 +6,10 @@ var environment = require('./environment');
 // used to prevent webpack from resolving requires on node libs
 var node = {require: require};
 
+var assign = environment.platform === 'node'
```

Comment:
I think this can be simplified to 

```js
var assign = require('object-assign');
```

since this library can be used in both node.js and the browser. That may also prevent some weird situations in the future if some day we want to use `assign` for other purposes as well but in the browser it's set to null.